MalApi/MalAppInfoXml.cs

```diff
@@ -194,6 +202,100 @@ public static MalUserLookupResults Parse(XDocument doc)
             return results;
         }
 
+        public static MalUserLookupResults ParseMangaResults(XDocument doc)
+        {
+            Logging.Log.Trace("Parsing XML.");
+
+            XElement error = doc.Root.Element("error");
+            if (error != null && (string)error == "Invalid username")
+            {
+                throw new MalUserNotFoundException("No MAL list exists for this user.");
+            }
+            else if (error != null)
+            {
+                throw new MalApiException((string)error);
+            }
+
+            if (!doc.Root.HasElements)
+            {
+                throw new MalUserNotFoundException("No MAL list exists for this user.");
+            }
+
+            XElement myinfo = GetExpectedElement(doc.Root, "myinfo");
+            int userId = GetElementValueInt(myinfo, "user_id");
+            string canonicalUserName = GetElementValueString(myinfo, "user_name");
```

Comment:
This first section of the method is duplicated from ParseAnimeResults. A way to avoid the duplication would be to add another constructor to MalUserLookupResults that takes both an anime list and a manga list and have one function in MalAppInfoXml that looks for both "anime" XML elements and "manga" XML elements.

MalUserLookupResults should then have an empty list for anime or manga instead of null.